### PR TITLE
Increase cooldown to 10 seconds; Update readme; Corrected Grammar

### PR DESCRIPTION
**Pull request by infinotiver**
So here is another pull request :)

## What changed?
* All basic `economy` commands now have a cooldown of `10` seconds (increased from earlier `2` seconds)
* Readme Updated
   * Updated link texts for images (`Here for img` --> `Click here for img`)
   * Added a new contributors section with images of all contributors 
* @AyushSehrawat YOU SHOULD learn basic english (Changed `Echo - A economy...` to `Echo - An economy...`) 

## Up next?
* @AyushSehrawat you killed grammar many a times, I can see that in the readme too lol
* Switch to `python-dotenv` instead of `data.json`

Just a question, do you want me to develop this bot as more of a full fledged feature rich economy bot or like a tutorial or starter for economy bots?
Also, please consider releasing a new version of `eco-bot`

### CODE REVIEW

**Comment by AyushSehrawat** [comment]
> All basic economy commands now have a cooldown of 10 seconds (increased from earlier 2 seconds)

Switch to 5 instead except the places i mentioned

> @AyushSehrawat YOU SHOULD learn basic english (Changed Echo - A economy... to Echo - An economy...)

My english is good, that was written in hurry as a guide on how to setup mongoDB for beginners (and also a note to me when i started my journey with mongoDB)

> Just a question, do you want me to develop this bot as more of a full fledged feature rich economy bot or like a tutorial or starter for economy bots?

This is a tutorial / starter bot for people, you can add more commands related to economy if you want

> Also, please consider releasing a new version of eco-bot

Will see, not really needed though since people usually clone repository instead of downloading code from version

> Switch to python-dotenv instead of data.json

Why not do in 1 pull request ?

**Comment by infinotiver** [comment]
> except the places i mentioned

Where did you mention the places?
___

> This is a tutorial / starter bot for people, you can add more commands related to economy if you want

Alright, so it is not for production ig, I'll add more economy related commands but that would be simpler, thanks for clarifying

**Comment by AyushSehrawat** [comment]
> > except the places i mentioned
> 
> Where did you mention the places?
> 
> > This is a tutorial / starter bot for people, you can add more commands related to economy if you want
> 
> Alright, so it is not for production ig, I'll add more economy related commands but that would be simpler, thanks for clarifying

Mentioned in the review to add cooldown on specific commands


It should run in production but it's mainly for starters

**Comment by infinotiver** [comment]
Can't view your review

**Comment by infinotiver** [comment]
@AyushSehrawat updated the cooldowns, 
- Balance has cool down of `2` seconds
- Rob command has cool down of `3600` seconds
- Except for `beg` and `gamble` (`10` seconds cool down) cmds, all other commands have cool down of `5` seconds

**Comment by infinotiver** [comment]
Please merge the branch now

**Review by AyushSehrawat — APPROVED** [review]

File: cogs/economy.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ async def balance(self, ctx, user: discord.Member = None):
         await ctx.send(embed=embed)
 
     @commands.command(aliases=["wd"])
-    @cooldown(1, 2, BucketType.user)
+    @cooldown(1, 10, BucketType.user)
```
[the remark this answers is not in the document]
Comment (infinotiver) [reply cogs/economy.py]: Now I am able to see your review idk why

**Pull request merged**